.travis.yml

```diff
@@ -7,4 +7,7 @@ install:
   - "pip install -r requirements.txt"
   - "pip install -r requirements.dev.txt"
 # command to run tests
-script: pep8 .
+script: 
+  - "pep8 ."
+  - "mkdir instance && cp config.py ./instance"
```

Comment:
I think we do. Otherwise this happened: https://travis-ci.org/freifunk-berlin/ca.berlin.freifunk.net/jobs/88609164#L288

```
FileNotFoundError: [Errno 2] Unable to load configuration file (No such file or directory): '/home/travis/build/freifunk-berlin/ca.berlin.freifunk.net/instance/config.py'
```